### ISSUE TITLE
Unable to enable full-screen [WORKAROUND FOUND, see comments]

### ISSUE THREAD

**Issue by Leffdy** [TheLegendOfMataNui/game-issues#274]
**Remember to add the following information:**

**Build Version:**
Rebuilt, October 26th 2019. 

**Steps to Reproduce:**

Game toggles into full, borderless upon boot and stays that way for 4-5 seconds before reverting to windowed, taking up a fourth of my monitor at most. What do?

**Comment by rexthemouse** [comment]
I have the same issue!
It's in the beta version as well as the rebuilt. I can't seem to even play the game, the mouse is forced to the far left! :c

Edit:
Oh you just press Alt+Enter to switch between windowed and fullscreen. Dur.

Just letting you guys know so you don't figure it out for yourselves after fiddling with the Config menu for an hour like this dummy did =w=;

**Comment by gb115** [comment]
I'm having the same issue. Hitting alt enter puts it in fullscreen, but still doesn't render the full size, even after changing the config file

**Comment by Vahkiti** [comment]
Is this still relevant as of 1.0.2? I imagine it would be, but figured I'd check in.

**Comment by KaiCodeTraveler** [comment]
> Is this still relevant as of 1.0.2? I imagine it would be, but figured I'd check in.

Yep. Just tried to play the Rebuild version, from the exe and the launcher, with and without the patch, it's still here.

Moreover, the issue make for my part my cursor to be locked on the top of the screen, and Alt+Enter doesn't fix it neither.
Tries with Alt+Tab, but it make another issue: the cursor inside and outside the game become invisible, or the mouse unusable anymore.

Did the fact that the patch make a new iteration of the game can be an issue maybe ?

**Comment by Palmiton049** [comment]
Just downloaded the rebuilt version and faced this issue. The fullscreen change to windowed can be fixed by changing the configs in dgVoodoo.conf, setting fullscreen from false to true:

`FullScreenMode=true`

Although this was fixed, the mouse lock on the far left side of the screen persisted. Just like the previous comment, Alt+Enter and Alt+Tab don't fix the issue. I tested that on both Multimonitor and Single monitor, both scenarios had the same mouse lock issue.

**Comment by Palmiton049** [comment]
Update: user Magi on Litestone's Discord found out the mouse stuck issue can be fixed by editing the Bionicle.ini file and changing the Windowed property to "0".
![imagem_2023-06-19_194253429](https://github.com/TheLegendOfMataNui/game-issues/assets/105246753/6cc97fab-1769-4988-8d75-d59a2ced9751)

**Comment by Manji-Marauder** [comment]
Changed these lines in the dgVoodoo.conf file and set resolution in the Bionicle.ini 

FullScreenMode=true

ScalingMode=stretch_ar

**Comment by Palmiton049** [comment]
I tried changing the ScalingMode line from centered_ar to stretch_ar, like the last comment, and it just made the game revert to the 640x480 original resolution (image 1). Maintaining the stretch_ar in the .conf file but changing the .ini resolution from the original resolution to my monitor's resolution (2K) made the UI elements not scale, retaining their original resolution and tucked in the upper left corner. (image 2)

Image 1
<img width="640" height="480" alt="Image" src="https://github.com/user-attachments/assets/72cfaa62-132e-43af-b33b-b75625eb34b7" />
Image 2
<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/6a4b8029-21b2-46de-be6d-01faa75ea753" />

The combination that has the best results for me is leaving the resolution on the .ini file in 853x480 (640x480 but in 16:9 aspect ratio) and leaving ScalingMode=centered_ar, as seen below (image 3)

Image 3
<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/088b7c4b-1212-47e6-9e20-493033308f25" />
